CHANGELOG.md

```diff
@@ -4,6 +4,35 @@ All notable changes to [Earthly](https://github.com/earthly/earthly) will be doc
 
 ## Unreleased
 
+## v0.7.0-rc1 - 2022-10-12
```

Comment:
Did you mean to use a January date in here?

```suggestion
## v0.7.0-rc1 - 2023-01-13
```

Reply:
We'll update this when we rewrite the release notes prior to the actual release -- there'll be a few more PRs before then.